member-service/src/main/java/com/hedvig/memberservice/commands/SetFraudulentStatusCommand.kt

```diff
@@ -0,0 +1,32 @@
+package com.hedvig.memberservice.commands
+
+import com.hedvig.memberservice.aggregates.FraudulentStatus
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+
+class SetFraudulentStatusCommand private constructor(
+    @TargetAggregateIdentifier
+    val memberId: Long,
+    val fraudulentDescription: String,
+    val token: String
+) {
+    var fraudulentStatus = FraudulentStatus.NOT_FRAUD
+
+    constructor(
+        memberId: Long,
+        fraudulentStatus: String?,
+        fraudulentDescription: String,
+        token: String
+    ) : this(
+        memberId,
+        fraudulentDescription,
+        token
+    ) {
+        if (fraudulentStatus != null) {
+            try {
+                this.fraudulentStatus = FraudulentStatus.valueOf(fraudulentStatus)
+            } catch (e: IllegalArgumentException) {
+                this.fraudulentStatus = FraudulentStatus.NOT_FRAUD
+            }
+        }
+    }
+}
```

Comment:
Since this is sent from hope and shouldn't be wrong we may as well break if we f up and have a default for when we don't send it I think
```suggestion
class SetFraudulentStatusCommand private constructor(
    @TargetAggregateIdentifier
    val memberId: Long,
    val fraudulentDescription: String,
    val token: String,
    val fraudulentStatus = FraudulentStatus.NOT_FRAUD
}
```